src/q7/matrix-INMA2380/exercises/ch1.tex

```diff
@@ -205,256 +296,290 @@ \section{Matrix' algebra}
-    we have that
+    Let $B$ be a square matrix of dimension $n$ commuting with $A = \diag\{a_1,\dots,a_n\}$.
+    We then have that
     \[
       B A =
-      \begin{pmatrix}
+      \begin{bmatrix}
         b_{11} a_1 & \dots & b_{1n} a_n \\
         \vdots & \ddots & \vdots \\
         b_{n1} a_1 & \dots & b_{nn} a_n \\
-      \end{pmatrix}
+      \end{bmatrix}
       =
-      \begin{pmatrix}
+      \begin{bmatrix}
         b_{11} a_1 & \dots & b_{1n} a_1 \\
         \vdots & \ddots & \vdots \\
         b_{n1} a_n & \dots & b_{nn} a_n \\
-      \end{pmatrix}
-      = A B
+      \end{bmatrix}
+      = A B.
     \]
     We thus need $b_{ij} a_j = b_{ij} a_i$ for every $i \neq j$
     and since $a_i \neq a_j$ for $i \neq j$, $B$ has to be diagonal itself.
   \end{description}
 \end{solution}
 
 \exo{1}
+Show that for every matrix \(A \in \C^{m \times n}\), the matrices \(AA^\top\) and \(A^\top A\) are symmetric, and the matrices \(AA^*\) and \(A^*A\) are Hermitian.
+
 \begin{solution}
-  Using the properties $(AB)^T = B^TA^T$ and $(AB)^* = B^*A^*$,
+  Using the properties $(AB)^\top = B^\top A^\top$ and $(AB)^* = B^*A^*$,
   \begin{align*}
-    (AA^T)^T & = (A^T)^TA^T\\
-             & = AA^T\\
-    (A^TA)^T & = A^T(A^T)^T\\
-             & = A^TA\\
+    (AA^\top)^\top & = (A^\top)^\top A^\top\\
+             & = AA^\top,\\
+    (A^\top A)^\top & = A^\top(A^\top)^\top\\
+             & = A^\top A,\\
     (AA^*)^* & = (A^*)^*A^*\\
-             & = AA^*\\
+             & = AA^*,\\
     (A^*A)^* & = A^*(A^*)^*\\
              & = A^*A.
   \end{align*}
 \end{solution}
 
 \exo{1}
+Show that for a square matrix \(A \in \C^{n \times n}\), the matrix \(A + A^\top\) is symmetric, the matrix \(A + A^*\) is Hermitian, the matrix \(A - A^\top\) is antisymmetric and the matrix \(A - A^*\) is anti-Hermitian.
+
 \begin{solution}
   We have
   \begin{align*}
-    (A + A^T)^T & = A^T + A\\
-                & = A + A^T\\
-    (A - A^T)^T & = A^T - A\\
-                & = -(A - A^T)\\
+    (A + A^\top)^\top & = A^\top + A\\
+                & = A + A^\top,\\
+    (A - A^\top)^\top & = A^\top - A\\
+                & = -(A - A^\top),\\
     (A + A^*)^* & = A^* + A\\
-                & = A + A^*\\
+                & = A + A^*,\\
     (A - A^*)^* & = A^* - A\\
                 & = -(A - A^*).
   \end{align*}
 \end{solution}
 
 \exo{2}
+Show that every complex matrix can be written as the sum of a symmetric matrix and an antisymmetric matrix, and as the sum of a Hermitian matrix and an anti-Hermitian matrix.
+
 \begin{solution}
   We simply notice that
   \begin{align*}
-    A & = \frac{A + A^T}{2} + \frac{A - A^T}{2}\\
+    A & = \frac{A + A^\top}{2} + \frac{A - A^\top}{2}\\
       & = \frac{A + A^*}{2} + \frac{A - A^*}{2}.
   \end{align*}
 \end{solution}
 
 \exo{1}
+Show that the Kronecker product satisfies
+\[
+(A \otimes B)^* = A^* \otimes B^*.
+\]
+
 \begin{solution}
   It is shown by
```

Comment:
Add `\allowdisplaybreaks` between the lines so that `align` allows page breaks.

Reply:
I reworded it instead.